server/src/main/java/com/paypal/selion/grid/matchers/SeLionSauceCapabilityMatcher.java

```diff
@@ -17,19 +17,54 @@
 
 import java.util.Map;
 
+import org.apache.commons.lang.StringUtils;
 import org.openqa.grid.internal.utils.CapabilityMatcher;
-import org.openqa.grid.internal.utils.DefaultCapabilityMatcher;
+
+import com.paypal.selion.proxy.SeLionSauceProxy;
 
 /**
- * This is a {@link CapabilityMatcher} based extension of {@link DefaultCapabilityMatcher} that nullifies the 
- * capabilities of the internal {@link DefaultCapabilityMatcher} which resides in the Grid.
- * This is essentially done to ensure that all of the {@link CapabilityMatcher} capabilities are delegated
- * to the one that dwells in the Sauce Labs cloud.
+ * Used in conjunction with a grid that has a {@link SeLionSauceProxy} connected. <br>
+ * <br>
+ * A simple {@link CapabilityMatcher} which checks for required information in the requested desired capabilities but
+ * does not perform any match against node capabilities. Matches against node capabilities will be performed by the
+ * Sauce labs cloud when {@link SeLionSauceProxy} forwards the session.
  * 
  */
-public class SeLionSauceCapabilityMatcher extends DefaultCapabilityMatcher {
-    @Override
-    public boolean matches(Map<String, Object> nodeCapability, Map<String, Object> requestedCapability) {
+public class SeLionSauceCapabilityMatcher implements CapabilityMatcher {
+    private static final String SAUCE_USER_NAME_CAPABILITY = "sauceUserName";
+    private static final String SAUCE_API_KEY_CAPABILITY = "sauceApiKey";
+    private static final String ACCESS_KEY_CAPABILITY = "accessKey";
```

Comment:
Super trivial but can you flip the var order to be consistent, SAUCE_USER_NAME_CAPABILITY SAUCE_API_KEY_CAPABILITY USERNAME_CAPABILITY ACCESS_KEY_CAPABILITY.  Helps me see whats going on a bit quicker